blog/2015/08/gsoc-finished.html

```diff
@@ -0,0 +1,46 @@
+---
+layout: blog
+root: ../../..
+author: Raniere Silva
+title: "GSoC 2015 Finished"
+date: 2015-08-28
+time: "10:00:00"
+category: ["Announcements"]
+---
+<!-- start excerpt -->
+<p>
+  And this year edition of
+  <a href="https://www.google-melange.com/gsoc/homepage/google/gsoc2015">Google Summer of Code</a> (GSoC)
+  finished.
+  We are very happy with the outcome of the work of the three students that
+  worked under the <a href="http://numfocus.org/">NumFOCUS</a> umbrella that we helped to coordinate.
+</p>
+<!-- end excerpt -->
+<p>
+  The projects were:
+</p>
+<ul>
+  <li><a href="https://insertinterestingnamehere.github.io/posts/gsoc-concluding-thoughts.html">Cython API for DyND</a>,</li>
+  <li><a href="http://piotr.banaszkiewicz.org/blog/2015/08/15/amy-update-6/">Enhance Amy, a workshop-management platform for Software Carpentry</a>,</li>
+  <li><a href="https://juliaquantum.github.io/news/2015/08/GSoC2015-Quantum-MCWF-method-and-QuDynamics">JuliaQuantum: Framework for solvers</a>.</li>
```

Comment:
@rgaiacs just a small edit. We have a wrap up post which has been updated, it would be great if the link were the following : http://juliaquantum.github.io/news/2015/08/GSoC2015-Wrap-up-and-Outlook/

Thank you for support all along the GSoC period !